Wrapping/Generators/Python/CMakeLists.txt

```diff
@@ -887,6 +887,11 @@ macro(itk_wrap_simple_type_python wrap_class swig_name)
     ADD_PYTHON_CONFIG_TEMPLATE("vector" "std::vector" "vector${swig_name}" "${cpp_name}< ${template_params} >")
   endif()
 
+  if("${cpp_name}" STREQUAL "itk::DTITubeSpatialObjectPoint")
```

Comment:
Could this long list of nearly-identical code be replaced by a for loop iterating over a list of spatial object types?

Reply:
Great idea.   Replaced with foreach loops